inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -130,6 +130,12 @@ public final class Constants {
                             + "is used extract database name from the raw binary data, "
                             + "this is only used in the multiple sink writing scenario.");
 
+    public static final ConfigOption<Boolean> SOURCE_MULTIPLE_ENABLE =
+            ConfigOptions.key("source.multiple.enable")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Whether migrate multiple databases");
```

Comment:
Maybe it is better to call 'Whether enable multiple source'?

Reply:
Now there is only one source, It should be appropriate to Call 'Which enable migrate multiple databases'.